packages/coinstac-decentralized-algorithm-integration/src/decentralized/computation-error-handled/index.js

```diff
@@ -11,6 +11,15 @@ module.exports = {
     cmd: 'python',
     args: ['./handled-error.py'],
     verbose: true,
+
+    /**
+     * @todo Remove once covariate inputs aren't required.
```

Comment:
Trying to notate all covariates-related stuff that should’t be required for computations/declarations. We can address and revert when we address backlog.